### PR TITLE
Create configuration documents for Service Catalog

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Add configuration documents for Service Catalog 
- Remove one line from the HB configuration document
- Fix formatting in the Application Broker architecture document

**Related issue(s)**

### CODE REVIEW

File: docs/service-catalog/05-03-catalog-crds.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| **controllerManager.resources.requests.memory** | Defines requests for memory resources. | `20Mi` |
+| **controllerManager.resources.limits.cpu** | Defines limits for CPU resources. | `100m` |
+| **controllerManager.resources.limits.memory** | Defines limits for memory resources. | `30Mi` |
+| **controllerManager.verbosity** | Defines log severity level. The possible values range from 0-10. | `10` |
+| **webhook.resources.requests.cpu** | Defines requests for CPU resources. | `100m` |
+| **webhook.resources.requests.memory** | Defines requests for memory resources. | `20Mi` |
+| **webhook.resources.limits.cpu** | Defines limits for CPU resources. | `100m` |
+| **webhook.resources.limits.memory** | Defines limits for memory resources. | `30Mi` |
+| **webhook.verbosity** | Defines log severity level. The possible values range from 0-10. | `10` |
+| **controllerManager.brokerRelistIntervalActivated** | Specifies whether or not the controller supports a --broker-relist-interval flag. If this is set to true, brokerRelistInterval will be used as the value for that flag. | `true` |
```
Comment (bszwarc) [inline docs/service-catalog/05-03-catalog-crds.md]:
```suggestion
| **controllerManager.brokerRelistIntervalActivated** | Specifies whether or not the controller supports a `--broker-relist-interval` flag. If this is set to `true`, brokerRelistInterval will be used as the value for that flag. | `true` |
```

File: docs/service-catalog/05-04-etcd-stateful.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Etcd-stateful sub-chart
+type: Configuration
+---
+
+To configure the Etcd-stateful sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
Comment (bszwarc) [inline docs/service-catalog/05-04-etcd-stateful.md]:
```suggestion
To configure the `etcd-stateful` sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
The formatting of the sub-chart according to what I found here: https://kyma-project.io/docs/1.2/components/service-catalog/#details-service-catalog-backup-and-restore

File: docs/service-catalog/05-05-podpreset.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: PodPreset sub-chart
+type: Configuration
+---
+
+To configure the PodPreset sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
Comment (bszwarc) [inline docs/service-catalog/05-05-podpreset.md]:
```suggestion
To configure the `PodPreset`sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```

File: docs/service-catalog/05-03-catalog-crds.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: Catalog CRDs sub-chart
+type: Configuration
+---
+
+To configure the Catalog CRDs sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
Comment (bszwarc) [inline docs/service-catalog/05-03-catalog-crds.md]:
```suggestion
To configure the `Catalog CRDs` sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```

File: docs/service-catalog/05-01-service-catalog.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+title: Service Catalog chart
+type: Configuration
+---
+
+To configure the Service Catalog chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
Comment (bszwarc) [inline docs/service-catalog/05-01-service-catalog.md]:
```suggestion
To configure the `Service Catalog` chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```

File: docs/service-catalog/05-02-catalog.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+title: Catalog sub-chart
+type: Configuration
+---
+
+To configure the Catalog sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
Comment (bszwarc) [inline docs/service-catalog/05-02-catalog.md]:
```suggestion
To configure the `Catalog` sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
not sure if it should be `Catalog` or `catalog` as usually chart names are not capitalized (unless I guess it's a proper name, like Service Catalog which is always capitalized)🤔

File: docs/service-catalog/05-04-etcd-stateful.md (hunk 2 above)
Comment (klaudiagrz) [reply docs/service-catalog/05-04-etcd-stateful.md]: Discussed

File: docs/service-catalog/05-05-podpreset.md (hunk 3 above)
Comment (klaudiagrz) [reply docs/service-catalog/05-05-podpreset.md]: Discussed

File: docs/service-catalog/05-03-catalog-crds.md (hunk 4 above)
Comment (klaudiagrz) [reply docs/service-catalog/05-03-catalog-crds.md]: Discussed

File: docs/service-catalog/05-01-service-catalog.md (hunk 5 above)
Comment (klaudiagrz) [reply docs/service-catalog/05-01-service-catalog.md]: Discussed

File: docs/service-catalog/05-02-catalog.md (hunk 6 above)
Comment (klaudiagrz) [reply docs/service-catalog/05-02-catalog.md]: Discussed

**Pull request merged**